stdlib/public/core/SwiftifyImport.swift

```diff
@@ -34,6 +39,9 @@ public enum _SwiftifyInfo {
     /// object past the lifetime of the function.
     /// Parameter pointer: index of pointer in function parameter list.
     case nonescaping(pointer: _SwiftifyExpr)
+    /// Can express lifetime dependencies between inputs and outputs of a function.
+    /// 'from' is the input on which the output 'to' depends.
```

Comment:
This comment took a while for me to parse. Perhaps more precise names for the parameters would help? Maybe something like `boundedBy` or `dependsOn` instead of `from `. `pointer` instead of `to` would be more consistent with the other cases (even if `std::span` isn't strictly a pointer).